Pages/Shared/_Layout.cshtml

```diff
@@ -81,11 +86,11 @@
         <div class="container" >
             <div class="row" >
 
-                <div class="col" style="left">
+                <div class="col" style="left:50px">
```

Comment:
In the _Layout.cshtml file there was a warning bug for 
"Missing a colon (':') between the property and value in the "(property) : (value)" declaration."
I have made the value additions to the C# Code.The warning has been resolved.This improves the reliability and performance of the Code.


Sources
https://www.ais.com/best-practices-starting-a-new-c-project/

Reply:
Linked stylesheets are much preferred over inline.